internal/spdxlicense/generate/generate_license_list_test.go

```diff
@@ -144,3 +145,50 @@ func TestFindLicenseVersion(t *testing.T) {
 		})
 	}
 }
+
+func TestReplaceDeprecatedLicenses(t *testing.T) {
```

Comment:
Question: Should this also test the case where a license is deprecated but no other license should be used to replace it?

Reply:
Good call, adding it